src/api/routes/index.ts

```diff
@@ -0,0 +1,18 @@
+import routeData from './data.json';
+
+export interface IRoute {
+  id: string;
+  name: string;
+  description: string;
+  // The id of the stops in order for the route
+  stops: string[];
+  // An array of arrays of schedules, so this should be
+  // the number of mins from midnight as our schedule is repeated daily
+  // e.g. [ [0, 100, 200, 200], [20, 400, 500] ]
```

Comment:
Why did you choose seconds from midnight as the time format? To make the eventual algorithm easier? 

I am wondering when the schedule changes significantly, if the task of updating the schedule might be too complex. Maybe the source schedule could be stored in a 24hh:mm or 12hh:mm and there is a script somewhere along the way to convert it to minutes from midnight to make life easier for the programmer, or perhaps there is a good time library out there that could be used for time queries (there must be...)

Reply:
I chose mins from midnight because seconds would be a bit too granular for what we need. As most of the schedule is just incremental in minutes. But I wanted to store it as a number so that we can do some calculations, e.g. which one is earlier, or min and max, etc

Good point on the conversion script. It shouldn't be too hard to do that. Actually would be cool if we can just read a CSV with our given format and then just parse through it and it will replace that data.